examples/gogo/command/src/main/java/org/osgi/enroute/examples/gogo/command/ServiceMulti.java

```diff
@@ -0,0 +1,39 @@
+
+package org.osgi.enroute.examples.gogo.command;
+
+import java.util.Hashtable;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceRegistration;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+
+@Component(immediate = true)
+public class ServiceMulti {
+
+    private final ServiceRegistration<Object> srv;
```

Comment:
This entire class should be removed. It is not a good pattern for a DS component to register itself in its activate method like this. There are occasions when it is necessary to do this, but this is not one of them. Providing this in an example will confuse users.

Reply:
> to register itself in its activate method

The component does not register itself but another newly created service.
I just used a component to trigger the registration instead of using a bundle activator.

Reply:
Even so, this is not something that we want to encourage as a pattern in an example for new users.